XCast/RtXcastConnector.cpp

```diff
@@ -309,20 +309,36 @@ int RtXcastConnector::applicationStateChanged( string app, string state, string
         LOGINFO(" xdialCastObj is NULL ");
     return status;
 }//app && state not empty
-void RtXcastConnector::enableCastService(string friendlyname,bool enableService)
+void RtXcastConnector::enableCastService(bool enableService)
```

Comment:
empty line before function could be added

Reply:
fixed